conf/muchos.props.example

```diff
@@ -19,9 +19,12 @@ cluster_type = ec2
 # Cluster user name (install command will SSH to cluster using this user)
 # Leave default below if launching cluster in AWS
 cluster_user = centos
-# Cluster base directory where install/ & tarballs/ directories are created
-# Leave default below if launching cluster in AWS
-cluster_basedir = /home/centos
+# Cluster user group
+cluster_group = %(cluster_user)s
+# Cluster user home directory
+user_home = /home/%(cluster_user)s
+# Install directory where Hadoop, Accumulo, etc will be installed
+install_dir = %(user_home)s/install
```

Comment:
Its nice to be able to set this.  Have you tried running muchos with it changed to something else?